velox/vector/VectorPool.h

```diff
@@ -69,19 +69,23 @@ class VectorPool {
 /// NOTE: the context object must provide 'releaseVector' vector such as
 /// EvalCtx.
 template <class Context>
-class ScopedVectorPtr {
+class VectorRecycler {
  public:
-  explicit ScopedVectorPtr(Context& context) : context_(context) {}
-  ~ScopedVectorPtr() {
+  explicit VectorRecycler(VectorPtr& vector, Context& context)
```

Comment:
Would it make sense to pas a VectorPool instead of a context?

Reply:
Yeah, good point!